src/lua/luaLoggerBinding.c

```diff
@@ -896,19 +896,23 @@ static int lua_get_virtual_channel(lua_State *ls)
          * Numbers can be passed in as 123 or "123" in lua.  So handle
          * that case first
          */
-        VirtualChannel *vc;
         if (lua_isnumber(ls, 1)) {
+        		VirtualChannel *vc;
                 vc = get_virtual_channel(lua_tointeger(ls, 1));
+                if (vc) {
```

Comment:
Same here with indentation.  Its messed up.  Please fix so we can keep this codebase clean